packages/office-ui-fabric-react/src/components/ScrollablePane/ScrollablePane.base.tsx

```diff
@@ -427,11 +515,108 @@ export class ScrollablePaneBase extends BaseComponent<IScrollablePaneProps, IScr
     const { contentContainer } = this;
 
     if (contentContainer) {
-      this._stickies.forEach((sticky: Sticky) => {
-        sticky.syncScroll(contentContainer);
+      // sync Sticky scroll if contentContainer has scrolled horizontally
+      if (this._scrollLeft !== contentContainer.scrollLeft) {
+        this._scrollLeft = contentContainer.scrollLeft;
+        this._stickies.forEach((sticky: Sticky) => {
+          sticky.syncScroll(contentContainer);
+        });
+      }
+      if (this._scrollTop !== contentContainer.scrollTop) {
+        this._userInteractionStarted = true;
+        this._scrollTop = contentContainer.scrollTop;
+        this._notifyThrottled();
+      }
+    }
+  };
+
+  private _sortBasedOnOrder(): boolean {
```

Comment:
This method seems unneeded.

Reply:
_sortBasedOnOrder() has the logic which decides if sorting has to be done based on 'order' prop for Sticky. I find _sortBasedOnOrder() to be more readable than using this.props.optimizeForPerformance.